scapy/main.py

```diff
@@ -199,8 +201,9 @@ def list_contrib(name=None):
 def update_ipython_session(session):
     """Updates IPython session with a custom one"""
     try:
+        global get_ipython  # defined externally by ipython / jupyter
         get_ipython().user_ns.update(session)
-    except:
+    except Exception:
```

Comment:
This change is unnecessary for this PR. Could you remove it?

Reply:
“We do enforce PEP8 for new code.”